src/components/BestProductList.jsx

```diff
@@ -0,0 +1,25 @@
+import ItemsPage from "../pages/ItemsPage";
+import BestProductCard from "./BestProductCard";
+
+function ProductList({ products, itemsPerPage }) {
+  return (
+    <ul className="grid grid-cols-1 tablet:grid-cols-2 pc:grid-cols-4 gap-8 tablet:gap-16 pc:gap-24 max-w-344 tablet:max-w-695 pc:max-w-1200 m-auto">
+      {products.slice(0, itemsPerPage).map((product) => {
+        const { id, name, images, price, favoriteCount } = product;
```

Comment:
💬 여담
아래 코드도 동일동작합니다~

```suggestion
      {products.slice(0, itemsPerPage).map(({ id, name, images, price, favoriteCount }) => {
```